installer/dockerfile/webhook-manager/Dockerfile

```diff
@@ -15,8 +15,13 @@
 
 # The base image is created via `Dockerfile.base`, the base image is cached
 # since the required packages change very rarely.
+FROM golang:1.17 AS builder
+WORKDIR /go/src/volcano.sh/
+ADD . volcano
+RUN cd volcano && make vc-webhook-manager
+
 FROM volcanosh/vc-webhook-manager-base:1.3.1
```

Comment:
Then change here to link current volcanosh/vc-webhook-manager-base instead of

Reply:
or just first publish a new `volcanosh/vc-webhook-manager-base` with multi-arch support: such as `volcanosh/vc-webhook-manager-base:1.5.1`, then we don't need to add  webhook-manager-base-image deps for `images:`